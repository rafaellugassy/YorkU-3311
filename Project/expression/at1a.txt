restart

quotient
analyze
simplify

restart

quotient
int_value (5)
int_value (2)
analyze
simplify 