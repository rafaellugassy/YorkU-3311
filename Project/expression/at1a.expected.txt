  Expression currently specified: ?
  Report: Expression is initialized.
->restart
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->quotient
  Expression currently specified: (? / nil)
  Report: OK.
->analyze
  Expression currently specified: (? / nil)
  Report: Error (Expression is not yet fully specified).
->simplify
  Expression currently specified: (? / nil)
  Report: Error (Expression is not yet fully specified).
->restart
  Expression currently specified: ?
  Report: OK.
->quotient
  Expression currently specified: (? / nil)
  Report: OK.
->int_value(5)
  Expression currently specified: (5 / ?)
  Report: OK.
->int_value(2)
  Expression currently specified: (5 / 2)
  Report: OK.
->analyze
  Expression currently specified: (5 / 2)
  Report: (5 / 2) is type-correct.
->simplify
  Expression currently specified: (5 / 2)
  Report: 2
